docs/site/examples.md

```diff
@@ -105,15 +95,64 @@ examples:
     links:
     - language: powershell
       url: "/tree/master/examples/knative/powershell/kn-ps-horizon-login-slack"
-  - title: VMware Cloud Gateway Notification (Slack)
+
+  - title: Distributed Virtual Switch configuration enforcement
+    usecases:
+    - item: remediation
+    id: kn-pcli-vds-config
+    description: Function to enforce `MTU` values on a distributed virtual switch. Any `MTU` changes will be intercepted by the function and reset to the desired value.
+    links:
+    - language: powershell
+      url: "/tree/master/knative/powercli/kn-pcli-vds-config"
+
+  - title: Distributed Virtual Switch portgroup configuration enforcement
+    usecases:
+    - item: remediation
+    id: kn-pcli-vds-pg-config
+    description: Function to enforce the `Notify Switches` value on a distributed virtual switch portgroup. Any changes to the `Notify Switches` value will be intercepted by the function and reset to the desired value.
+    links:
+    - language: powershell
```

Comment:
nit: We have a separate section for `powercli`. Do you want to change all of the affected examples accordingly?

Reply:
Fixed